changelog/7760.breaking.rst

```diff
@@ -0,0 +1,2 @@
+:meth:`sunpy.map.GenericMap.contour` now uses ContourPy instead of scikit-image for improved performance and consistency with :meth:`sunpy.map.GenericMap.draw_contours`.
```

Comment:
The changelog entry needs to warn that the returned contours will likely be different (in order, if not in values)